tests/casp/oci_install.pm

```diff
@@ -21,7 +21,7 @@ sub run() {
     send_key 'alt-p';    # partitioning
     assert_screen 'prepare-hard-disk';
     send_key 'alt-b';    # back
-    if (check_screen 'error-small-disk') {
+    if (check_screen 'error-small-disk', 3) {
```

Comment:
I will remove the check because bug is fixed and this only creates false positive now - https://openqa.suse.de/tests/924108#step/oci_install/4